samples/VS2015/SpecFlow 2.1.0/net461/Sample.Website.Tests/project.json

```diff
@@ -15,7 +15,7 @@
 	},
 
 	"tools": {
-		"SpecFlow.NetCore": "1.0.0-rc4"
+		"SpecFlow.NetCore": "1.0.2"
```

Comment:
that's just for my testing, sorry. Nuget keeps caching the tool so I have to update versions all the time. Do you know of a workaround?